ocs_sample_library_preview/Security/AccessControlEntry.py

```diff
@@ -57,9 +57,13 @@ def fromJson(content: dict[str, str]):
 
         if 'AccessType' in content:
             result.AccessType = AccessType(content['AccessType'])
+        else:
```

Comment:
Are these two `else` statements necessary? Do we ever get back null/undefined values for these fields? It makes sense that we default to `Allowed` and `none`, but I also feel like `None` and `None` would get the point across.

Reply:
As far as I am aware, you should never get back null/undefined values for the AccessType field. The default access type is Allowed and it will not be returned if it is the default so this else statement is required for everything to function correctly. I did the same for AccessRights since I was getting issues when it was set to ```None``` when calling ```value.name``` in the ```toDictionary``` function of the ```AccessRights``` class.

Reply:
I guess it's fine, it just breaks our normal pattern where we don't usually manually assign the default values if the server didn't give us one. @derekendres and @kelseyduffy any thoughts here?

Reply:
@cbardini  what do you mean by "it will not be returned if it is the default"? All the calls I've made return either "Allowed" or "Denied".

Reply:
@kelseyduffy It only shows the parameter if it is "Denied" or if the header ```Accept-Verbosity: verbose``` is included

Reply:
got it, just tested that out with modheader and I see what you mean. If that's the behavior, then I think we have to just set it to Allowed if it's missing

Reply:
Yeah it would be that or setting the verbose header by default